titles/admin-rhdh/title-admin.adoc

```diff
@@ -37,3 +37,6 @@ include::assemblies/assembly-admin-templates.adoc[leveloffset=+1]
 
 // techdocs plugin
 include::assemblies/assembly-techdocs-plugin.adoc[leveloffset=+1]
+
+// RHDH Operator deployment
+include::assemblies/assembly-rhdh-deployment.adoc[leveloffset=+1]
```

Comment:
Consider including directly the module (no assembly required here).

Reply:
Just for my edification: it is better to include this module in the `title-admin` file now, and not create an assembly for future content. The assembly and moving of this file into it would be up to the writer tasked with additional deployment content.